cmd/iamctl/minify.go

```diff
@@ -1,30 +1,95 @@
 package main
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"sort"
 	"strings"
+	"text/template"
 )
 
+// compressionPrefixes defines a list of action prefixes in the policy
+// that are going to be compressed using wildcards.
+var compressionPrefixes = map[string]string{
+	"ec2:Describe":                  "ec2:Describe*",
+	"elasticloadbalancing:Describe": "elasticloadbalancing:Describe*",
+}
+
+func generateIAMPolicy(inputFilename, output, pkg string) {
+	tmpl, err := template.New("").Parse(filetemplate)
+	if err != nil {
+		panic(err)
+	}
+
+	jsFs, err := ioutil.ReadFile(filepath.Join(assetDir, inputFilename))
```

Comment:
Remind me again why we take the input as `assetDir+inputFilename`. Can we not specify the relative path of the input file directly?

Reply:
Fixed